frontend/src/components/sidebar/index.tsx

```diff
@@ -2,15 +2,22 @@ import React from 'react';
 import sidebarStyles from './sidebar.module.scss';
 import sidebarOptions, { SidebarOption } from '../../constants/sidebarOptions';
 
-const getButtons = (options: SidebarOption[]) =>
+const getButtons = (options: SidebarOption[], selctedOption: String) =>
   options.map((o) => (
-    <button type="button" className={sidebarStyles.menuOption} key={o.name}>
+    <button
+      type="button"
+      className={`${sidebarStyles.menuOption} ${
+        o.name === selctedOption ? sidebarStyles.selectedMenuOption : ''
+      }`}
+      key={o.name}>
       {o.name}
     </button>
   ));
 
-const Sidebar = () => (
-  <div className={sidebarStyles.container}>{getButtons(sidebarOptions)}</div>
+const Sidebar = ({ selctedOption }: { selctedOption: string }) => (
```

Comment:
Are the curly braces neccessary around the `selectedOption` and `selectedOption: string`?

Reply:
I believe so, because the function is called with a "props object" which I am deconstructing